SQL/Archive/17.1/2017-06-06_CreateRandomJWTKey.sql

```diff
@@ -0,0 +1 @@
+UPDATE Config SET Value=(SELECT FLOOR(RAND()*(999999999-1+1))+1) WHERE Value = "S3cret";
```

Comment:
I don't think it's a good idea to have this patch if the key is blacklisted. 

With a blacklisted key, you can be sure that it won't work after updating. With this, the key generated may still be guessable. (I doubt MySQL's `RAND()` function is cryptographically secure, and even if it were using an integer as a key for a JWT token should be trivial to crack using brute-force)